packages/driver/src/cypress/error_messages.ts

```diff
@@ -2103,13 +2117,11 @@ export default {
 
           The previous URL you visited was:
 
-            > '{{previousUrl}}'
+            > '${args.previousUrl.origin}'
 
           You're attempting to visit this URL:
 
-            > '{{attemptedUrl}}'
-
-          You may need to restructure some of your test code to avoid this problem.`,
+            > '${args.attemptedUrl.origin}'`,
         docsUrl: 'https://on.cypress.io/cannot-visit-different-origin-domain',
```

Comment:
should we flex this docs url depending on if they have the feature flag enabled?, Send them to `cy.origin` if it is enabled?

Reply:
I was hoping this doc would be updated to inform the user why they received this error and link to `cy.origin`.

Reply:
ok, this is fine for now, lets double check with DX next week.